tooling/message-generators/src/main.rs

```diff
@@ -158,22 +158,27 @@ fn make_generators<'a, X>(seed: Option<&[u8]>, len: usize) -> Generators
 where
     X: BbsCiphersuite<'a>
 {
-
+    const EXPAND_LEN: usize = 48; //TODO make a property of X or calculate?
     let default_seed = &X::generator_seed();
-    let seed = seed.unwrap_or(default_seed);
+    let seed = seed.unwrap_or(default_seed); //TODO remove parameter and always use default one?
```

Comment:
```suggestion
    let seed = seed.unwrap_or(default_seed);
```
This is just regarding the TODO ;P In the spec there is the option of supplying an issuer specific seed and not the default one. There is some discussion around this and it may change in the future, but currently this is here to allow for this (note that the `signer_specific_generators` will pass the Issuer's pk as the seed).